errata.html

```diff
@@ -1,55 +0,0 @@
----
-layout: default
-title: Errata
```

Comment:
Do we still need this?  I couldn't find it being ported over, but this is a big diff so I might have missed it

Reply:
In the v1 website we have both a [releases page](https://glacierprotocol.org/releases.html) and [errata page](https://glacierprotocol.org/errata.html), but I think that information should live the closest to the actual Glacier release/package as possible. We should also avoid having duplicate information that can easily get out of sync.

So the strategy that I used in this PR is that everything related to a protocol release (zip package, release notes, errata) should be tracked in a singe place - [the GH releases page for the protocol](https://github.com/GlacierProtocol/GlacierProtocol/releases).

That design decision lead to two things:
* The site doesn't allow you to download the Glacier zip package. Instead, it always sends you to the GH releases page.
* There is no release notes or errata pages. If you check the [GH releases page](https://github.com/GlacierProtocol/GlacierProtocol/releases), you'll see that the information is now there.

Reply:
the rationale makes sense, but there are a few links in the website to `https://glacierprotocol.org/errata` and `https://glacierprotocol.org/errata.html` that now point to the 404 page.  Can we change it to reference the releases page?

Reply:
Good catch! I've updated the PR to replace all links to `/errata` and `/releases` by the GH releases url.